pkgs/applications/audio/vcv-rack/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv, fetchFromGitHub, fetchurl, pkgconfig
+, alsaLib, curl, glew, glfw, gtk2-x11, jansson, libjack2, libXext, libXi
+, libzip, rtaudio, rtmidi, speex }:
+
+let
+  # We need newer versions for some of the dependencies. They are normally
```

Comment:
Have you tried using the existing versions in nixpkgs? It would be preferable for those to be used if possible.

Reply:
The Nix package for rtaudio indeed works. I think I missed that because I was building against the stable Nix channel.

We still need a newer version for glfw though. There hasn't been an upstream release yet. :/